sourcefinder/image.py

```diff
@@ -25,23 +26,6 @@
 
 logger = logging.getLogger(__name__)
 
-#
```

Comment:
Nice to see how the config object reduces the number of lines of code and makes it more readable.